### ISSUE TITLE
Sign up process

### ISSUE THREAD

**Issue by viniciusban** [glancer000/recruit-landing-page#4]
The MVP user sign up flow is:

1. User fill their data.
1. User fill their profile.
1. _Recruit_ shows a welcome screen.

## User account data

- username
- password
- email

Rules:

- No duplicate username or email.
- A really long password field. Any character allowed (improves security).

## User profile

Pick only one:

- frontend
- backend
- ui
- whatever

## Questions

1. Should we allow user to login with GitHub account?

**Comment by glancer000** [comment]
Yes, I think we should allow people to manually sign up, use GitHub and potentially use Google to sign in

**Comment by viniciusban** [comment]
I thought about signing up with GitHub account because their repo should be hosted there, right?

**Comment by glancer000** [comment]
One of the points made last night was about data collection. If they were to sign into the site with Facebook, we would be able to collect a lot of basic information from users.

The point of view is that we need to keep data on the platform as much as possible, so Facebook login would allow for the basic information on a person to be auto populated.

On another note, I don't think Firebase has Github connected as an easy login option. I could be wrong there.

**Comment by glancer000** [comment]
Opps, I was wrong, Git Hub is on the list of option on there. So maybe it can be part of the options.

**Comment by viniciusban** [comment]
OK. We must make a decision to the MVP.

Which credentials user should use to sign up?

**Comment by glancer000** [comment]
I have made the sign up work with email right now. But since Firebase offers Facebook and Github, we can try both and see how it works out.
